src/plugins/data_views/common/data_views/data_views.ts

```diff
@@ -678,7 +678,7 @@ export class DataViewsService {
     );
 
     if (!savedObject.version) {
-      throw new SavedObjectNotFound('data view', id, 'management/kibana/dataViews');
```

Comment:
Reviewers: I'm not sure if anything depended on there being a link in the error, but it isn't rendering correctly when it's shown in the error toast notification. So I removed the link option from this erro.